libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -573,6 +575,10 @@ static ImagePredicate localMediaIdPredicate(String id) {
             return new ImagePredicate(id, "data-wpid");
         }
 
+        static ImagePredicate genericPredicate(String idName, String id) {
```

Comment:
No reason, really. I've modified this in another PR: https://github.com/wordpress-mobile/WordPress-Android/blob/issue/5217-stop-image-upload/libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java#L580. I could address it there, if you'd like.